ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/JavaCodeGenerator.java

```diff
@@ -289,6 +290,9 @@ protected Path toResponseParserPath(final String fileName) {
      */
     protected Template getResponseParserTemplate(final Ds3Request ds3Request) throws IOException {
         //TODO special case as necessary
```

Comment:
There are still more commands that might need special casing. Its not "needed", but the refactor isn't complete yet.